question/app.js

```diff
@@ -0,0 +1,153 @@
+//Q１ 変数
+let nickname = '倉ちゃん'
+let age = 27
+
+console.log('私ニックネームは'+nickname+'です。年齢は'+age+'です。');

```

Comment:
[must]
`+` の前後は空白をもうけてください！
```suggestion
console.log('私ニックネームは' + nickname + 'です。年齢は' + age + 'です。');

```

Reply:
失礼いたしました。修正しましたのでご確認お願いいたします。